client/src/AppSecondary.jsx

```diff
@@ -0,0 +1,283 @@
+
```

Comment:
A good portion of this code is duplicate of your App component. Two alternatives might be:

1. Separate the reservation form portion of this into its own component?
```
// app with form
<App>
  <ReservationForm />
</App>

// app without form
<App />
```

2. Reuse the App component, using a` secondary` prop and conditional logic for whether to display the form?
```
<App />
<App secondary />
```

Reply:
Yeah good call. I like the secondary logic idea cuz there are actually a lot of little differences. At some point I'm going to diff the code and figure out how to split it. I might also separate any unchanged functions into their own library that each imports.